src/main/scala/beam/router/RoutingWorker.scala

```diff
@@ -478,7 +484,8 @@ object RoutingWorker {
           0,
           unbecomeDriverOnCompletion = true
         )
-      )
+      ),
+      Some("R5")
```

Comment:
if it is not used in filtering or any kind of logic, I would use "Bushwalking" here as route type